R-packages/evalcast/R/manage_cards.R

```diff
@@ -66,11 +66,65 @@ intersect_locations <- function(cards) {
   cards %>% map(~ .x %>% filter(location %in% intersected_locations))
 }
 
+#' Aggregate cards from a list into a single unnested data frame
+#'
+#' @param list_of_cards List of prediction or evaluation cards.  See documentation for the outputs
+#'   of `evalcast::get_predictions()` and `evalcast::evaluate_predictions()` for the required
+#'   format.
+#' @return Data frame such that:
+#'   \item There is a one-to-one correspondence between rows of the output and the rows of
+#'     `card$forecast_distribution` across each `card` in `list_of_cards`.  That is to say, if each
+#'     card has 7 rows in its `forecast_distribution` and there are 3 such cards in
+#'     `list_of_cards`, the output will have 21 rows.
+#'   \item The columns of the output correspond to:
+#'     \itemize{
+#'       \item Columns of `card$forecast_distribution`
+#'       \item Columns of `card`
+#'       \item Attributes of `card`
+#'     }
+aggregate_cards <- function(list_of_cards) {
```

Comment:
I think you need to export this function.

Reply:
Done

Reply:
Sorry, I should have been more explicit. It needs `#' @export` on its own line above `aggregate_cards` or else the function won't be available to the package user.